tests/test-cases/demo-stacks.yaml

```diff
@@ -164,11 +164,6 @@ tests:
         - "./components/terraform/github/stargazers/main/providers.tf"
         - "./components/terraform/github/stargazers/main/variables.tf"
         - "./components/terraform/github/stargazers/main/versions.tf"
-        - "./components/terraform/infra/my-vpc1/main.tf"
-        - "./components/terraform/infra/my-vpc1/outputs.tf"
-        - "./components/terraform/infra/my-vpc1/providers.tf"
-        - "./components/terraform/infra/my-vpc1/variables.tf"
-        - "./components/terraform/infra/my-vpc1/versions.tf"
```

Comment:
If we are commenting it out above we should comment out here, since we intend to restore this.

Reply:
done